server/plugins/LFUGens.cpp

```diff
@@ -664,9 +676,12 @@ void LFTri_Ctor(LFTri* unit) {
     }
 
     unit->mFreqMul = 4.0 * unit->mRate->mSampleDur;
-    unit->mPhase = ZIN0(1);
+
+    double initPhase = unit->mPhase = sc_wrap((double)ZIN0(1), 0.0, 4.0);
```

Comment:
Can you make this a static_cast<double> please rather than a c cast.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es48-avoid-casts

Reply:
done!